docs/UserGuide.md

```diff
@@ -431,6 +442,7 @@ Format: `viewTasks`
 </div>
 
 #### 4.2.2 Add a task: `addTask`
+>>>>>>> 96cba0b589bd1ba2e5e8187baae09b5633709240
 
```

Comment:
I believe this needs to be removed? Seems to be a remainder of a merge conflict.